cmake/thirdparty/FindCUB.cmake

```diff
@@ -11,7 +11,9 @@ find_path(CUB_INCLUDE_DIRS
   NAMES cub/cub.cuh
   HINTS
     ${CUB_DIR}/
-    ${CUB_DIR}/include)
+    ${CUB_DIR}/include
+    ${CUDA_TOOLKIT_ROOT_DIR}/include
+    ${CUDA_TOOLKIT_ROOT_DIR}/include/thrust/system/cuda/detail)
```

Comment:
Is this still an actively used path? I know it was there when they only distributed enough for thrust to work, but would have thought the versions with a full copy of cub wouldn't use it anymore.

Reply:
The ${CUDA_TOOLKIT_ROOT_DIR}/include path is where I found cub in cuda 11.
The ${CUDA_TOOLKIT_ROOT_DIR}/include/thrust/system/cuda/detail is where I found cub in cuda 9, 10. This install did work for the examples and tests but it could be only a partial implementation, I didn't look at it that closely.